app/tests/v1/test_user_model.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from app.api.v1.models.users import UserModel
+from instance.db_config import connection
+
+class TestUserModel(unittest.TestCase):
+    pass
+    # def test_user_database_creation(self):
```

Comment:
Avoid presenting commented block of code for reviews, have a clean code

Reply:
Sure. I will factor that out